updating/understanding-upgrade-channels-release.adoc

```diff
@@ -19,6 +19,11 @@ ifndef::openshift-origin[]
 
 If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the `oc adm upgrade channel` command in the OpenShift CLI to configure an empty channel. This configuration can be helpful if, for example, a cluster has restricted network access and there is no local, reachable upgrade recommendation service.
 
+[WARNING]
+====
```

Comment:
I think that you might want to use "might" instead of "can" in this sentence. 

IBM SG regarding "can": Use to indicate ability. Use instead of "could". To indicate possibility, use "might". To indicate permission, use "may".

Reply:
Addressed